lib/ObstacleHandler.cpp

```diff
@@ -31,6 +31,11 @@ std::string ObstacleInfo::getJsonKey() const
 	return identifier;
 }
 
+std::string ObstacleInfo::getModScope() const
+{
+	return "core";
```

Comment:
`ObstacleInfo` is missing `modScope` but I think obstacles can be provided by mods. Is it intentional or should I introduce it and initialize in `loadFromJson`?

Reply:
Yeah, sounds like a bug.
While not 100% safe - fixing this one might reveal bugs someplace else (if any), but still a good idea to do.